cluster-operator/src/test/java/io/strimzi/operator/cluster/TestUtils.java

```diff
@@ -66,21 +69,34 @@ public static Map<String, String> containerEnvVars(Container container) {
     public static void maybeAddHttpProxyEnvVars(List<EnvVar> envVars) {
         if (System.getenv(ClusterOperatorConfig.HTTP_PROXY) != null) {
             envVars.add(new EnvVarBuilder()
-                .withName(ClusterOperatorConfig.HTTP_PROXY)
-                .withValue(System.getenv(ClusterOperatorConfig.NO_PROXY))
-                .build());
+                    .withName(ClusterOperatorConfig.HTTP_PROXY)
+                    .withValue(System.getenv(ClusterOperatorConfig.NO_PROXY))
+                    .build());
         }
         if (System.getenv(ClusterOperatorConfig.HTTPS_PROXY) != null) {
             envVars.add(new EnvVarBuilder()
-                .withName(ClusterOperatorConfig.HTTPS_PROXY)
-                .withValue(System.getenv(ClusterOperatorConfig.HTTPS_PROXY))
-                .build());
+                    .withName(ClusterOperatorConfig.HTTPS_PROXY)
+                    .withValue(System.getenv(ClusterOperatorConfig.HTTPS_PROXY))
+                    .build());
         }
         if (System.getenv(ClusterOperatorConfig.NO_PROXY) != null) {
             envVars.add(new EnvVarBuilder()
-                .withName(ClusterOperatorConfig.NO_PROXY)
-                .withValue(System.getenv(ClusterOperatorConfig.NO_PROXY))
-                .build());
+                    .withName(ClusterOperatorConfig.NO_PROXY)
+                    .withValue(System.getenv(ClusterOperatorConfig.NO_PROXY))
+                    .build());
         }
     }
+
+    public static Pod podWithName(String name) {
+        return podWithNameAndAnnotations(name, Collections.emptyMap());
+    }
+
+    public static Pod podWithNameAndAnnotations(String name, Map<String, String> annotations) {
+        return new PodBuilder()
+                .withNewMetadata()
+                    .withName(name)
+                    .withAnnotations(annotations)
+                .endMetadata()
+                .build();
+    }
```

Comment:
There are many different places where similar logic is used. So it makes sense to have it shared. But you should go through all the tests doing this and update them to use this method as well.

Reply:
I already moved this out from the manual rolling update test class. These methods were declared just there. I moved in this common TestUtils to be reused by my tests. If we want to use the same somewhere else, I don't think it's part of this PR.

Reply:
Anyway, I guess I can live with this if needed.